frontend/src/text_resources.js

```diff
@@ -347,10 +347,10 @@ let LOCALIZE = new LocalizedStrings({
         orgChartInstructions:
           "Below is a tree view of the organization chart. Once selected, you can use the arrow keys to navigation, expand, and collapse information.",
         backgroundInformation: {
-          title: "Background Information"
+          title: "Overview"
```

Comment:
Are these @caleybrock's changes from one of the last PRs?
Did you merge in master?

Reply:
merged in master: 29179ff (that was weird)